File: pgm/ABC_Java/ABC_312_A.java
import java.io.*;
// import java.math.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S = reader.readLine();
        final List<String> list =
            List.of ("ACE", "BDF", "CEG", "DFA", "EGB", "FAC", "GBD");
        final String Yes = "Yes";
        final String No = "No";
        System.out.println (list.stream().anyMatch (s -> S.equals (s)) ? Yes
                                                                       : No);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_312_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"

fun main() {
    val S: String = readln()
    val arr: Array<String> = arrayOf("ACE", "BDF", "CEG", "DFA", "EGB", "FAC", "GBD")
    if (S in arr) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}